### ISSUE TITLE
Can not use 'taylor_var' and 'make_Taylor_model'

### ISSUE THREAD

**Issue by Cfather** [JuliaIntervals/TaylorModels.jl#94]
```
julia> t = taylor_var(2, 0, -0.5..0.5)
ERROR: UndefVarError: taylor_var not defined
Stacktrace:
 [1] top-level scope at REPL[2]:1

julia> t1 = make_Taylor_model(exp, 3, 0, -0.5..0.5)
ERROR: UndefVarError: make_Taylor_model not defined
Stacktrace:
 [1] top-level scope at REPL[3]:1
```

I tried to run the examples but failed. These two functions 'taylor_var' and 'make_Taylor_model' seem not defined in this repo. I wonder whether there's any pkg I haven't include or import?

**Comment by lbenet** [comment]
Thanks for reporting, and sorry for the confusion of having that file which is related to an old version of  the package. 

@mforets is addressing this, updating the examples file in #95. Some tests are not passing yet, which I will look up to see what can we do (only in Julia 1.1). In any case, I recommend you to take a look on the proposed changes (https://github.com/JuliaIntervals/TaylorModels.jl/pull/95/files).

**Comment by mforets** [comment]
Close?

**Comment by Cfather** [comment]
Thank you for the example!

Marcelo Forets <notifications@github.com> 于2021年1月18日周一 上午10:04写道：

**Comment by lbenet** [comment]
Closed by #95
